pkg/osvscanner/osvscanner.go

```diff
@@ -356,27 +366,30 @@ func scanImage(r reporter.Reporter, path string) ([]scannedPackage, error) {
 // within to `query`
 func scanLockfile(r reporter.Reporter, path string, parseAs string, transitiveAct TransitiveScanningActions) ([]scannedPackage, error) {
 	var err error
-	var parsedLockfile lockfile.Lockfile
-
-	f, err := lockfile.OpenLocalDepFile(path)
-
-	if err == nil {
-		// special case for the APK and DPKG parsers because they have a very generic name while
-		// living at a specific location, so they are not included in the map of parsers
-		// used by lockfile.Parse to avoid false-positives when scanning projects
-		switch parseAs {
-		case "apk-installed":
-			parsedLockfile, err = lockfile.FromApkInstalled(path)
-		case "dpkg-status":
-			parsedLockfile, err = lockfile.FromDpkgStatus(path)
-		case "osv-scanner":
-			parsedLockfile, err = lockfile.FromOSVScannerResults(path)
-		default:
-			if !transitiveAct.Disabled && (parseAs == "pom.xml" || filepath.Base(path) == "pom.xml") {
-				parsedLockfile, err = extractMavenDeps(f, transitiveAct)
-			} else {
-				parsedLockfile, err = lockfile.ExtractDeps(f, parseAs)
+
+	var inventories []*extractor.Inventory
+
+	// special case for the APK and DPKG parsers because they have a very generic name while
+	// living at a specific location, so they are not included in the map of parsers
+	// used by lockfile.Parse to avoid false-positives when scanning projects
+	switch parseAs {
+	case "apk-installed":
+		// inventories, err := apkinstalled.Extractor{}.Extract(context.Background(), &si)
```

Comment:
any reason why we keep this commented line around? Can you add a comment / link to a relevant bug?

Reply:
Ah that comment should not be there, removed.